common/src/main/java/org/keycloak/common/util/ServerCookie.java

```diff
@@ -33,7 +33,9 @@ public class ServerCookie implements Serializable {
     private static final String tspecials2 = "()<>@,;:\\\"/[]?={} \t";
 
     public enum SameSiteAttributeValue {
```

Comment:
Plan to do that, but that's a follow-up as it needs some changes to the bits that sets cookies, otherwise I'd have to temporarily convert between the two ;)